demo/App.vue

```diff
@@ -14,60 +14,42 @@
 </template>
 
 <script>
-import moment from "moment";
-import "../dist/vuetify-daterange-picker.css";
-import DateRange from "../";
+import subDays from 'date-fns/sub_days';
+import format from 'date-fns/format';
+import '../dist/vuetify-daterange-picker.css';
+import DateRange from '../';
 export default {
   components: {
-    DateRange
+    DateRange,
   },
   data() {
     return {
       range: [],
       dateRangeOptions: {
-        startDate: moment()
-          .subtract(7, "days")
-          .format("YYYY-MM-DD"),
-        endDate: moment().format("YYYY-MM-DD"),
-        format: "MM/DD/YYYY",
+        startDate: format(subDays(new Date(), 7), 'YYYY-MM-DD'),
```

Comment:
Why isn't prettier breaking the chained methods? For me it does that and it looks pleasing to my eyes that way. Can we please keep that type of formatting?

Reply:
I have this in my editor settings
```json
  "prettier.printWidth": 120,
```

How much `printWidth` do you use?

We can have a prettier config for this repo like this https://github.com/coderplex/coderplex/blob/develop/.prettierrc so `prettier` can use this file instead of our editor config.

Reply:
We can have a prettier config. That would be nice.. but i dont have any and still by default all chained methods are broken into lines. Wondering why THAT isn't happening for you

Reply:
Can we set the print width to 80 please? otherwise it kind of looks haywire to me.

Reply:
Okay sure

Reply:
BTW `format(subDays(new Date(), 7), 'YYYY-MM-DD'),` this is not chained function calls, this has nested function calls. Maybe that's why it's not wrapping into new lines as chained functional calls